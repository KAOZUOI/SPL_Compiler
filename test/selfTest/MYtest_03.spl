int add(int a, int b) {
  return a + b;
}

int minus(int c, int d) {
  return c - d;
}

int main() {
  int result = add(5, 7.5);
  float result = minus(5, 7.5);
  return 0;
}
