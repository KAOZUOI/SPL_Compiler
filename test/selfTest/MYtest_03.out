Error type 9 at Line 10: the function "add" is not applicable for arguments.
Error type 3 at Line 11: a variable is redefined in the same scope "result".
Error type 9 at Line 11: the function "minus" is not applicable for arguments.
Error type 5 at Line 11: unmatching types appear at both sides of the assignment operator (=) 